src/npm-resolve.ts

```diff
@@ -1,13 +1,124 @@
+import { createDeno } from './deno.ts';
 import { PluginConfig } from './types.ts';
 
-export default function npmResolve({}: PluginConfig) {
+export default function npmResolve(config: PluginConfig) {
+  // TODO(bartlomieju): both plugins should use the same instance, or maybe
```

Comment:
It's fine to create two separate ones. The cache instances are passed in from the plugin config in mod.ts.